.azure-pipelines-gh-pages.yml

```diff
@@ -28,6 +28,16 @@ jobs:
         parameters:
           cmake_args: ""
 
+      - script: |
+          set -ex
+          python3.8 -m venv env
+          source env/bin/activate
+          pip install wheel
+          pip install -U -r doc/requirements.txt
+          pip install -U -e ./python
+          sphinx-build -b linkcheck doc build/html
```

Comment:
Can we include this in the existing single-version build, rather than as a separate step?

Reply:
As discussed, I couldn't find a way to do this in one step because `linkcheck` is a different builder altogether.